packages/data-layer/src/queries.ts

```diff
@@ -337,6 +337,12 @@ export const getRoundByIdAndChainId = gql`
       strategyId
       strategyAddress
       strategyName
+      applications(first: 1000, filter: { status: { equalTo: APPROVED } }) {
```

Comment:
please check where this query is used before changing it like this, remember that the data layer is shared between all the dapps, I would suggest to make a new query like getRoundByIdAndChainIdWithApprovedApplications or similar

Reply:
@boudra  In other places where this query is being used, applications are not being used only basic round data which is why im re using the same query with additional data.

Reply:
exactly, now places where they don't need applications are over fetching the graphql for applications, querying more tables in the database

Reply:
creating a new query is cheap for us, overfetching data is not cheap for some of our users. agree with @mo here @bhargavaparoksham

Reply:
This is used in builder and might slow us down a bit. Would recommend creating a new query